### ISSUE TITLE
No ConvTranspose layer in your code？

### ISSUE THREAD

**Issue by 1hell0the1others8** [LeeJunHyun/Image_Segmentation#65]
Hello,
I found that there is ConvTrans layer in your network architecture. But i can not find ConvTranspose layer in the code you supported. 
Could you solve my confusion?
Thank you!

class up_conv(nn.Module):
    def __init__(self,ch_in,ch_out):
        super(up_conv,self).__init__()
        self.up = nn.Sequential(
            nn.Upsample(scale_factor=2),
            nn.Conv2d(ch_in,ch_out,kernel_size=3,stride=1,padding=1,bias=True),
		    nn.BatchNorm2d(ch_out),
			nn.ReLU(inplace=True)
        )

    def forward(self,x):
        x = self.up(x)
        return x

**Comment by fengbear** [comment]
Upsample
